js/ui/html_editor/modules/base.js

```diff
@@ -0,0 +1,25 @@
+import Quill from 'devextreme-quill';
+
+let BaseHtmlEditorModule = class EmptyModule {};
```

Comment:
```suggestion
import EmptyModule from './empty';

let BaseHtmlEditorModule = EmptyModule;
```
And so on for other modules